wordpress-mobile/WordPress-Android
# Site Title overlap on Tablets #16817

Fixes #16817
Site Title overlap on Tablets

Solution
In `MySiteFragment.kt` in the `MySiteFragmentBinding.updateCollapsibleToolbar` function for a tablet all the values ie `appBarLayout.totalScrollRange and (verticalOffset)` are zero. Therefore the `updateCollapsibleToolbar` always shows the title on the `collapsingToolbar` and hides the  `siteInfoCard`. The solution was to check if the device `appBarLayout.totalScrollRange` is `0` and if it is, call `updateCollapsibleToolbar` with a value of one and set the percentage to 100.

To test:

- Log into the app on a Tablet Device
- Make sure you have more than 1 site connected and you have one with a long title

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Visual testing of the UI changes

3. What automated tests I added (or what prevented me from doing so)
No need to add any more tests

PR submission checklist:


Before UI
**Bug** [Loom Video](https://www.loom.com/share/c5071b2ca8a345208a9033c22562155b)

![175532359-fbd82d89-00f8-4398-a4a0-864918d11b31](https://user-images.githubusercontent.com/2725300/183880730-b420c33b-5c81-4084-becb-5519a018f934.jpg)

After Fix UI 
![image](https://user-images.githubusercontent.com/2725300/185386916-e2f877b9-2771-46aa-99ca-61424e0d1969.png)

**On Scroll Up**

![image](https://user-images.githubusercontent.com/2725300/185387444-b70fc37e-3181-4f00-89b2-c14908a7a6fe.png)

Loom video [here](https://www.loom.com/share/cb24417151054033862730238fb80f58)
